cache.go

```diff
@@ -36,40 +36,47 @@ var (
 	setBufSize = 32 * 1024
 )
 
-type itemCallback func(*Item)
+const itemSize = int64(unsafe.Sizeof(storeItem[any]{}))
 
-const itemSize = int64(unsafe.Sizeof(storeItem{}))
+//type HashableKey interface {
+//	KeyToHash() (uint64, uint64)
+//}
+
+func Zero[T any]() T {
```

Comment:
can we not export this function and rename it to something else? Zero is a well defined node in Dgraph. Maybe call it zeroValue?